onnx/backend/test/report/coverage.py

```diff
@@ -114,7 +128,116 @@ def report_text(self, writer):  # type: (IO[Text]) -> None
             else:
                 attrs_column = 'No attributes'
             rows.append([op_cov.op_type, attrs_column])
+            passed.append(op_cov.op_type)
         writer.write(tabulate(
             rows,
             headers=['Operator', 'Attributes\n(name: #values)'],
             tablefmt='plain'))
+        if os.environ.get(str('CSVDIR')) is not None:
```

Comment:
Can we split this part into a separate function? seems the logic is independent, and probably we should add some comments about how to use it?